airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -251,10 +252,12 @@ def post_dag_run(dag_id, session):
         .first()
     )
     if not dagrun_instance:
-        dag_run = DagRun(dag_id=dag_id, run_type=DagRunType.MANUAL, **post_body)
-        session.add(dag_run)
-        session.commit()
-        return dagrun_schema.dump(dag_run)
+        try:
+            dag = current_app.dag_bag.get_dag(dag_id)
+            dag_run = dag.create_dagrun(run_type=DagRunType.MANUAL, state=State.RUNNING, **post_body)
```

Comment:
Why did we change this call to add a state here?

As of a recent-ish change only the Scheduler should ever put a DagRun in to RUNNING state, everyting else should create/put it in the QUEUED state.

Reply:
Cause `state` was a required parameter in that function. Will use `QUEUED`.